installer/Octopus.Tentacle.Installer/Product.wxs

```diff
@@ -140,16 +140,11 @@
     <!--
     Harden installation directory
     -->
-    <CustomAction Id="HardenEXE"
-                  Directory="INSTALLLOCATION"
-                  ExeCommand="&quot;[INSTALLLOCATION]Octopus.Tentacle.Hardener.exe&quot;"
-                  Execute="deferred"
-                  Return="ignore"
-                  Impersonate="no"
-                 />
-
+    <CustomAction Id="SetExecProp" Property="WixQuietExecCmdLine" Value='"[POWERSHELLEXE]" -NoProfile -NonInteractive -InputFormat None -File "[INSTALLLOCATION]Harden-InstallationDirectory.ps1"'></CustomAction>
+    <CustomAction Id="HardenInstallationDirectory" BinaryKey="WixCA" DllEntry="WixQuietExec" Execute="immediate" Return="ignore"/>
```

Comment:
I don't know if we want this `Return="ignore"` here - if it's included, then even if the harden script fails, the installation will still say it succeeded.

Reply:
In consultation with TSO we decided that we would make best effort to run the script, but it was not essential for a successful Tentacle install.